.github/workflows/build-slides.yml

```diff
@@ -3,7 +3,15 @@ name: Build LaTeX Slides
 on: [push, pull_request]
 
 jobs:
+  test_slides_change:
+    runs-on: ubuntu-latest
+    uses: tj-actions/verify-changed-files@v6.2
+    with:
+      files: |
+        packaging/slides/*
```

Comment:
Is that recursive? So does it still work when `packaging/slides/images/*` is changed?

Reply:
And I just realize that git is completely missing

Reply:
That's what I am trying to test.
At the moment recursive or not, no action start 😆